ui/src/assets/i18n/de.json

```diff
@@ -0,0 +1,1082 @@
+{
+  "commons": {
+    "id": "Id",
+    "ok": "Ok",
+    "of": "von",
+    "yes": "Ja",
+    "ofType": "vom Typ",
+    "refresh": "Aktualisieren",
+    "cancel": "Abbrechen",
+    "remove": "Entfernen",
+    "edit": "Editieren",
+    "close": "Schließen",
+    "loading": "lade",
+    "listedBelow": "unterhalb aufgeführt",
+    "areYouSure": "Sind Sie sicher?",
+    "setDefault": "Als Standard setzen",
+    "makeDefault": "Zum Standard machen",
+    "importFile": "Importiere eine Datei",
+    "selectFile": "Selektiere eine Datei",
+    "minDate": "Min Datum",
+    "maxDate": "Max Datum",
+    "actions": "Aktionen",
+    "restart": "Neustarten",
+    "update": "Updaten",
+    "deploy": "Ausrollen",
+    "undeploy": "Zurückziehen",
+    "destroy": "Zerstören",
+    "launch": "Starten",
+    "schedule": "Zeitplan",
+    "scaleInstance": "Skaliere Instanz(en)",
+    "description": "Beschreibung",
+    "status": "Status",
+    "clone": "Klonen",
+    "date": "Datum",
+    "name": "Name",
+    "type": "Typ",
+    "stop": "Stopp",
+    "cleanup": "Aufräumen",
+    "details": "Details",
+    "version": "Version",
+    "platform": "Plattform",
+    "properties": "Eingeschaften",
```

Comment:
Eigenschaften

Reply:
Found some other typos like this as well. 👍